build.csx

```diff
@@ -148,32 +150,33 @@ targets.Add(
     DependsOn("build", "testsDirectory"),
     () =>
     {
-        RunTests(unitTests);
-        RunDotNetTests(netstdUnitTestDirectories);
+        RunDotNetTests(unitTestDirectories);
+        RunTests(analyzerTestAssemblies);
     });
 
 targets.Add(
+    "analyzer",
+    DependsOn("build", "testsDirectory"),
+    () => RunTests(analyzerTestAssemblies));

```

Comment:
Already covered in `unit`, which I think is the better place for it. I think this target can be removed.